docs/pages/cloud-client-ip-restrictions.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Teleport Cloud Client IP Restrictions
+description: Block CIDR ranges from connecting to your cloud cluster.
```

Comment:
The description is a bit misleading because it describes the inverse of the feature.

The feature is an allow list, but the description sounds a lot more like a deny list.